web/views/relay_group_view.ex

```diff
@@ -1,36 +1,37 @@
 defmodule Cog.V1.RelayGroupView do
   use Cog.Web, :view
 
-  def render("relay_group.json", %{relay_group: relay_group}) do
+  alias Cog.V1.BundlesView
+  alias Cog.V1.RelayView
+
+  def render("relay_group.json", %{relay_group: relay_group}=params) do
     %{id: relay_group.id,
       name: relay_group.name,
-      relays: render_relays(relay_group.relays()),
-      bundles: render_bundles(relay_group.bundles()),
       inserted_at: relay_group.inserted_at,
       updated_at: relay_group.updated_at}
+    |> Map.merge(render_includes(params, relay_group))
   end
+
   def render("index.json", %{relay_groups: relay_groups}) do
-    %{relay_groups: render_many(relay_groups, __MODULE__, "relay_group.json")}
+    %{relay_groups: render_many(relay_groups, __MODULE__, "relay_group.json", include: [:relays, :bundles])}
   end
   def render("show.json", %{relay_group: relay_group}) do
-    %{relay_group: render_one(relay_group, __MODULE__, "relay_group.json")}
+    %{relay_group: render_one(relay_group, __MODULE__, "relay_group.json", include: [:relays, :bundles])}
   end
-  def render("relays.json", %{relay_group: relay_group}) do
-    %{relays: render_relays(relay_group.relays())}
+  def render("members.json", %{relay_group: relay_group}) do
+    %{relays: render_include(:relays, relay_group)}
   end
 
-  defp render_relays(relays) do
-    for relay <- relays do
-      %{id: relay.id,
-        name: relay.name}
-    end
+  defp render_includes(params, relay_group) do
+    Map.get(params, :include, [])
+    |> Enum.reduce(%{}, fn(inc, reply) -> Map.put(reply, inc, render_include(inc, relay_group)) end)
   end
 
-  defp render_bundles(bundles) do
-    for bundle <- bundles do
-      %{id: bundle.id,
-        name: bundle.name}
-    end
+  defp render_include(:bundles, relay_group) do
+    render_many(relay_group.bundles, BundlesView, "bundle.json", as: :bundle)
+  end
+  defp render_include(:relays, relay_group) do
+    render_many(relay_group.relays, RelayView, "relay.json", as: :relay)
```

Comment:
This will end up sending more data than the basics and if we aren't careful we can get into a data loop of sorts where the bundle includes the relays, which includes the bundles, which includes the relays, which [blah, blah, blah]... I like the `render_include`, but I think we may want to use it sparingly, so we don't get twisted.